### PR TITLE
Feature/gh 2210

### LINKED ISSUE #2210: Extract translations from page meta resolvers
Example PR how to do it:
https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/1777/files#diff-ec2077dcc817fa50f8f16be2159aaaf5

### PR DESCRIPTION
**Pull request by mszczepaniak**
Closes #2210

### CODE REVIEW

File: projects/core/src/checkout/services/cart-page-meta.resolver.ts [hunk 1]
```diff
@@ -32,13 +37,19 @@ export class CartPageMetaResolver extends PageMetaResolver
...
-    return this.cartService
-      .getActive()
-      .pipe(
-        map(cart =>
-          cart && cart.code ? `${page.title} (${cart.code})` : page.title
-        )
-      );
+    return this.cartService.getActive().pipe(
+      switchMap(cart =>
+        cart && cart.code
```
Comment (Platonn) [inline projects/core/src/checkout/services/cart-page-meta.resolver.ts]: Can we simplify it a little?
```
this.translation.translate('cartPage.heading', {
  title: page.title,
  code: cart && cart.code || ''
})
```

Comment (mszczepaniak) [reply projects/core/src/checkout/services/cart-page-meta.resolver.ts]: sure, done

File: projects/storefrontlib/src/translations/en/meta-resolver.en.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  checkoutPage: {
+    heading: 'Checkout {{count}} item',
+    heading_plural: 'Checkout {{count}} items',
+  },
+  cartPage: {
+    heading: '{{title}} ({{code}})',
+  },
+  searchPage: {
+    heading: '{{total}} result for {{query}}',
```
Comment (Platonn) [inline projects/storefrontlib/src/translations/en/meta-resolver.en.ts]: the variable must have name `count` to trigger the pluralization logic

Comment (mszczepaniak) [reply projects/storefrontlib/src/translations/en/meta-resolver.en.ts]: done

**Pull request merged**